### PR TITLE
[chore] prepare release v0.10.0

### PR DESCRIPTION
**Pull request by bacherfl**

### CODE REVIEW

**Review by odubajDT — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,60 @@
...
+    Converter `IsRootSpan()` returns `true` if the span in the corresponding context is root, that means its `parent_span_id` equals to hexadecimal representation of zero. In all other scenarios function returns `false`.
+- `pkg/ottl`: Emit traces for statement sequence executions to troubleshoot OTTL statements/conditions ([#33433](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33433))
+- `receiver/filelog`: Add filelog.container.removeOriginalTimeField feature-flag for removing original time field ([#33946](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33946))
+- `statsdreceiver`: Allow configuring summary percentiles ([#33701](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33701))
+
+</details>
+
+### 🚀 New components 🚀
+
+- `statsdreceiver`: Add statsd receiver to manifest (#272)
```
Comment (mowies) [inline CHANGELOG.md:58]: I think the PR number is not the right one here.
```suggestion
- `statsdreceiver`: Add statsd receiver to manifest (#229)
```

Comment (bacherfl) [reply CHANGELOG.md]: good catch, thanks!

**Review by mowies — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,60 @@
...
+- `receiver/filelog`: Add filelog.container.removeOriginalTimeField feature-flag for removing original time field ([#33946](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33946))
+- `statsdreceiver`: Allow configuring summary percentiles ([#33701](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33701))
+
+</details>
+
+### 🚀 New components 🚀
+
+- `statsdreceiver`: Add statsd receiver to manifest (#229)
+- `zipkinreceiver`: Added the zipkin receiver to the Dynatrace collector distro. (#225)
+
```
Comment (arminru) [inline CHANGELOG.md:60]: Should we move this section up to above the deps upgrade so it's more prominent?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,60 @@
...
+- `pkg/ottl`: Emit traces for statement sequence executions to troubleshoot OTTL statements/conditions ([#33433](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33433))
+- `receiver/filelog`: Add filelog.container.removeOriginalTimeField feature-flag for removing original time field ([#33946](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33946))
+- `statsdreceiver`: Allow configuring summary percentiles ([#33701](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33701))
+
+</details>
+
+### 🚀 New components 🚀
+
+- `statsdreceiver`: Add statsd receiver to manifest (#229)
+- `zipkinreceiver`: Added the zipkin receiver to the Dynatrace collector distro. (#225)
```
Comment (arminru) [inline CHANGELOG.md:59]:
```suggestion
- `statsdreceiver`: Add statsd receiver to manifest. (#229)
- `zipkinreceiver`: Added the zipkin receiver to the Dynatrace collector distro. (#225)
```
or
```suggestion
- `statsdreceiver`: Add statsd receiver to manifest (#229)
- `zipkinreceiver`: Added the zipkin receiver to the Dynatrace collector distro (#225)
```

**Review by arminru — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bacherfl) [reply CHANGELOG.md]: good point - yes i think those additions deserve to be placed more prominently - will adapt that

**Review by arminru — APPROVED** [review]

**Pull request merged**